booster/unit-tests/Test/Booster/Pattern/Rewrite.hs

```diff
@@ -172,6 +173,19 @@ testConf = do
             , terminalLabels = []
             }
 
+ignoreRulePredicateAndSubst :: RewriteResult Pattern -> RewriteResult Pattern
+ignoreRulePredicateAndSubst =
+    \case
+        RewriteBranch pre posts ->
+            RewriteBranch
+                pre
+                ( NE.map
+                    ( \(nextState, ruleMetadata) -> (nextState, ruleMetadata{rulePredicate = Predicate TrueBool, ruleSubstitution = mempty})
+                    )
+                    posts
+                )
+        other -> other
+
```

Comment:
Maybe we should instead keep the actual predicate and substitution results in the test cases? (hopefully they are not too large)

Reply:
Indeed it does make sense to stop ignoring them since they are stable now, done!